### ISSUE TITLE
radio group in feedback form not a11y compliant

### ISSUE THREAD

**Issue by dynamic11** [CDH-Studio/UpSkill#1671]
**Describe the bug**
The radio button group in the feedback modal is not screen reader friendly
should be replaced with dropdown

**To Reproduce**

1. Go to app footer
2. Click on 'report a bug'
3. Scroll down to tab to radio group
4. See screen reader not working correctly

**Expected behavior**
It should read all the options

**Screenshots**
![image](https://user-images.githubusercontent.com/11470442/135153753-c203af02-3317-49a7-90f0-066584563a22.png)
